phpunit.xml.dist

```diff
@@ -16,4 +16,14 @@
             </exclude>
         </whitelist>
     </filter>
+
+    <php>
+        <env name="SYMFONY_PHPUNIT_VERSION" value="8.5.9"/>
```

Comment:
as using the phpunit bridge `8` is enough that the newest phpunit bridge is always installed. `^8.5.9` would only be required when we require phpunit package in the composer.json.